From 880487b04a744138f62269c252b1dcdbb4a22afe Mon Sep 17 00:00:00 2001
From: Jani Lusikka <jani.lusikka@gmail.com>
Date: Fri, 25 Mar 2016 20:12:15 +0200
Subject: [PATCH 16/18] Fix the group permissions of the sdcard root.

If the root is not writable, we cannot create new files or folders there.

Change-Id: I3edad81f105bbeabc9dcd1f5322acea6fbfc63bb
---
 fs/Ext4.cpp | 2 +-
 fs/F2fs.cpp | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/fs/Ext4.cpp b/fs/Ext4.cpp
index 3c01650..6de0e41 100644
--- a/fs/Ext4.cpp
+++ b/fs/Ext4.cpp
@@ -166,7 +166,7 @@ status_t Mount(const std::string& source, const std::string& target, bool ro,
 
     if (portable && rc == 0) {
         chown(c_target, AID_MEDIA_RW, AID_MEDIA_RW);
-        chmod(c_target, 0755);
+        chmod(c_target, 0775);
     }
 
     if (rc && errno == EROFS) {
diff --git a/fs/F2fs.cpp b/fs/F2fs.cpp
index 26d41e9..d1e5bd9 100644
--- a/fs/F2fs.cpp
+++ b/fs/F2fs.cpp
@@ -77,7 +77,7 @@ status_t Mount(const std::string& source, const std::string& target,
 
     if (portable && res == 0) {
         chown(c_target, AID_MEDIA_RW, AID_MEDIA_RW);
-        chmod(c_target, 0755);
+        chmod(c_target, 0775);
     }
 
     if (res != 0) {
-- 
2.8.2

